src/app-layout/visual-refresh/drawers.tsx

```diff
@@ -222,7 +222,11 @@ function DesktopTriggers() {
     return 0;
   };
 
-  const { visibleItems, overflowItems } = splitItems(drawers ?? undefined, getIndexOfOverflowItem(), activeDrawerId);
+  const { visibleItems, overflowItems } = splitItems(
+    drawers?.filter(item => !!item.trigger) ?? undefined,
```

Comment:
4th time a comment about filtering this list

Reply:
It's quite straightforward to fix this for the classic and layout-toolbar themes. However, for the VR theme, it will be more challenging since this component consumes drawers from a common context. The only way to filter out such a prop is from within the component itself

Reply:
I think we should create separate `drawers` and `drawerTriggers` lists on the common context